app/controllers/admin/node_search_controller.rb

```diff
@@ -0,0 +1,41 @@
+module Admin
+  class NodeSearchController < ::Api::V3::ApiController
+    skip_before_action :load_context
+
+    def source_search
+      @q = Api::V3::Readonly::Dashboards::Source.
+        select('dashboards_sources_mv.name, dashboards_sources_mv.node_type, dashboards_sources_mv.id, countries.name as country_name, commodities.name as commodity_name').
+        joins('LEFT JOIN commodities ON commodities.id = dashboards_sources_mv.commodity_id').
+        joins('LEFT JOIN countries ON countries.id = dashboards_sources_mv.country_id').
+        ransack(params[:q])
```

Comment:
can we add `.order('dashboards_sources_mv.name')` and analogically in other search controllers?